pysidt/sidt.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, mol, value):
         self.mol = mol
         self.value = value
 
+    def __repr__(self) -> str:
+        return f"{self.mol.smiles} {self.value}"
```

Comment:
Does this construction work? I didn't think you could use fields in f-strings?

Reply:
Yeah I was able to use this without error. When you print a list of Datums, it shows up like this now:

```
[C=CC(CC)C(O)CCC -428.315919,
 OCCNCC(C#CC)CCO -557.485736,
 CNCCC(C=C(CC)C)N -463.726963,
 CC(NCC)COC=N -421.130946,
 CCCC(C#N)(NC)C -384.058996,
 CC(C#C)NC(=N)C -343.570257,
 CC(C)CC(O)C(=O)O -460.903049,
 NC=NCC(C(C)C)O -421.150567,
 CC(CNC(=O)C(=C)C)N -459.219893,
...
]
```